target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -16,7 +16,9 @@ do_sysinfo_octeon() {
 	"UBNT_E220"*)
 		name="erpro"
 		;;
-
+	"ITUS_SHIELD"*)
```

Comment:
What's the exact name?
Please move this up based on alphabetic sorting (and, sorry, please either remove the empty lines between also case members here or add them for your entry as well.

Reply:
machine=$(grep "^system type" /proc/cpuinfo | sed "s/system type.*: \(.*\)/\1/g")
returns ITUS_SHIELD (CN7020p1.2-1000-AAP)

Reply:
Okay, so your choice of using a wildcard seems good here.